src/components/QuranNav/index.js

```diff
@@ -0,0 +1,76 @@
+import React, { PropTypes } from 'react';
+import LinkContainer from 'react-router-bootstrap/lib/LinkContainer';
+import Navbar from 'react-bootstrap/lib/Navbar';
+import Nav from 'react-bootstrap/lib/Nav';
+import NavDropdown from 'react-bootstrap/lib/NavDropdown';
+import MenuItem from 'react-bootstrap/lib/MenuItem';
+import NavItem from 'react-bootstrap/lib/NavItem';
+import Image from 'react-bootstrap/lib/Image';
+const Header = Navbar.Header;
+const Collapse = Navbar.Collapse;
+const Toggle = Navbar.Toggle;
+import { connect } from 'react-redux';
+
+const styles = require('./style.scss');
+
+export const QuranNav = ({ user }) => (
+  <Navbar inverse fluid className={styles.nav}>
+    <Header>
+      <Toggle />
+    </Header>
+    <Collapse>
+      <Nav />
+      <Nav pullRight>
+        <LinkContainer to="/apps" data-metrics-event-name="IndexHeader:Link:Mobile">
+          <NavItem>
+            Mobile
+          </NavItem>
+        </LinkContainer>
+        <NavItem href="https://quran.zendesk.com/hc/en-us/articles/210090626-Development-help" target="_blank" data-metrics-event-name="IndexHeader:Link:Developer">
+          Developers
+        </NavItem>
+        <NavItem href="http://legacy.quran.com" data-metrics-event-name="IndexHeader:Link:Legacy">
+          Legacy Quran.com
+        </NavItem>
+        <LinkContainer to="/donations" data-metrics-event-name="IndexHeader:Link:Contribute">
+          <NavItem>
+            Contribute
+          </NavItem>
+        </LinkContainer>
+        <NavItem href="https://quran.zendesk.com/hc/en-us" data-metrics-event-name="IndexHeader:Link:Contact">
+          Contact us
+        </NavItem>
+        {
+          user &&
+            <NavDropdown
+              title={
+                <span className={styles.name}>
+                  <Image src={user.image} className={styles.image} circle />
+                  {user.firstName}
+                </span>
+              }
+              id="user-dropdown"
+            >
+              <LinkContainer
+                to="/profile"
+                data-metrics-event-name="IndexHeader:Link:Profile"
+              >
+                <MenuItem eventKey={3.1}>Profile</MenuItem>
+              </LinkContainer>
+              <MenuItem eventKey={3.3}>Logout</MenuItem>
+            </NavDropdown>
+        }
+      </Nav>
+    </Collapse>
+  </Navbar>
+);
+
+QuranNav.propTypes = {
+  user: PropTypes.object
```

Comment:
Once again I'm a big fan of `PropTypes.shape({})`

Although since this is repeated it may be a good idea to create a types folder with all your major object types Props in one location and all you can do is import them so it would be like this:

```javascript
import userType from '../types/user';

static propTypes = {
  user: PropTypes.shape(userType),
}
```